packages/babel-plugin/package.json

```diff
@@ -18,7 +18,6 @@
   ],
   "dependencies": {
     "@babel/helper-module-imports": "^7.16.7",
```

Comment:
this depends on `@babel/types` but I guess that it should be mostly OK since those are just factories for simple objects and they shouldn't rely on identity of any class or anything like that